src/app/home/stats/GenreChart.tsx

```diff
@@ -0,0 +1,26 @@
+import { getTranslations } from "next-intl/server";
+import { popularBookGenres } from "../marketPanel/search/searchConfig";
+import PieChartGraph from "@/app/statsGraphs/PieChartGraph";
+
+export default async function GenreChart({
+  categoryCounts,
+}: {
+  categoryCounts: Record<string, number>;
+}) {
+  const t = await getTranslations();
+  const labelToKey = Object.fromEntries(
+    popularBookGenres.map((g) => [g.label, g.key])
+  );
+
+  const translatedCategoryCounts = Object.entries(categoryCounts).reduce(
+    (acc, [label, count]) => {
+      const key = labelToKey[label];
+      const display = key ? t(`genre_${key}`) : label;
```

Comment:
What about defaults if we don't have the value? Probably a good idea to provide something instead of just using the translation key.